sem_covid/entrypoints/notebooks/topic_modeling/topic_modeling_wrangling/topic_visualizer.py

```diff
@@ -1,8 +1,77 @@
 
 from PIL.Image import Image
 
+import pyLDAvis
+import pyLDAvis.gensim_models
+
+import pandas as pd
+from gensim.models import LdaMulticore
 from wordcloud import WordCloud
 
+from sem_covid.entrypoints.notebooks.topic_modeling.topic_modeling_wrangling.lda_modeling import WordsModeling
+
+
+class TopicInformation(WordsModeling):
```

Comment:
do we need this class? could we fall back onto a functional implementation?

Reply:
just because it uses the same attributes as WordModeling